diffprivlib/tools/histograms.py

```diff
@@ -131,7 +131,7 @@ def histogram(sample, epsilon=1, bins=10, range=None, normed=None, weights=None,
                       "To ensure differential privacy, and no additional privacy leakage, the range must be "
                       "specified independently of the data (i.e., using domain knowledge).", PrivacyLeakWarning)
 
-    accountant.check(epsilon, 0)
+    BudgetAccountant.check(accountant, epsilon, 0)
```

Comment:
Why static method instead of instance method?

Reply:
That has since changed, but it's still a static method, in order to check that the passed argument is in fact a `BudgetAccountant`. This check now takes place within `load_default(acc)`.